Configuration/StandardSequences/python/Reconstruction_cff.py

```diff
@@ -59,9 +59,10 @@
                           standalonemuontracking*
                           trackingGlobalReco*
                           vertexreco)
-_globalreco_tracking_Phase1PU70 = globalreco_tracking.copy()
-_globalreco_tracking_Phase1PU70.replace(trackingGlobalReco, recopixelvertexing+trackingGlobalReco)
-eras.trackingPhase1PU70.toReplaceWith(globalreco_tracking, _globalreco_tracking_Phase1PU70)
+_globalreco_tracking_LowPU_Phase1PU70 = globalreco_tracking.copy()
```

Comment:
I see that you merged the names.
I'd rather have a copy-pasted ``` _globalreco_tracking_LowPU ``` and ``` _globalreco_tracking_Phase1PU70 ```

I guess, this assumes that trackingPhase1PU70 will forever remain the SLHC almost-run1-like scenario used in TP, right?
[I was hoping it will go away at some point.]

Imagine we need to define a more proper Phase1 for PU70 in a couple of years. These merged-name LowPU_Phase1PU70 will have to be split up.